src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig

```diff
@@ -0,0 +1,105 @@
+{% extends 'NoTimeToWaitBundle::base.html.twig' %}
+
+{% block title %}No Time to Wait 4: Initializing Initiatives{% endblock %}
+
+{% block body %}
+<h1><strong>No Time to Wait!</strong> *</h1>
+
+<h2>Initializing Initiatives</h2>
+
+The fourth annual No Time to Wait conference shall take place December 5th and 6th, 201 at the <a href="http://www.osaarchivum.org">Vera and Donald Blinken Open Society Archives</a> in downtown Budapest, Hungary. #nttw4
+
+<h2>What</h2>
+<p>A FREE two-day symposium focused on <em>open media</em>, <em>open standards</em>, and <em>digital audiovisual preservation</em> hosted by <a href="http://www.osaarchivum.org/">the Vera and Donald Blinken Open Society Archives</a> and <a href="{{ path('homepage') }}">MediaArea.net</a>.</p>
+
+<p>The event will feature presentations and discussion on topics such as:</p>
+
+<ul>
+  <li>developing collaborative initiatives to preserve media</li>
+  <li>efforts to extend capabilities and opportunties within archives</li>
+  <li>active open media standardization projects</li>
+  <li>seeking consensus in audiovisual preservation strategy</li>
+  <li>examination of open media use in film and video digitization</li>
+  <li>integration of open source tools into archival workflow</li>
+  <li>examples of cross-community collaboration and skill-sharing</li>
+  <li>developments in open media</li>
+  <li>and more</li>
+</ul>
+
+<p>Follow <a href="https://twitter.com/nttwconf">@nttwconf</a> for news and updates about No Time to Wait. Our recommended hashtag is <a href="https://twitter.com/hashtag/nttw4">#nttw4</a>.</p>
+<p>The conference will be livestreaming on the MediaArea YouTube channel <a href="http://www.youtube.com/c/MediaAreaNetOfficial">available here</a>.</p>
+
+<h2>When</h2>
+<p>Thursday, December 5, 2019 and Friday, December 6, 2019</p>
+<p>An optional workshop shall be held on December 4, 2019</p>
+
+<p>Opening: 09:00<br />
+Closing: 17:00</p>
+
+<p>The 2 day symposium includes drinks and lunch on site. We are seeking a sponsor (or two) to support a social dinner for Thursday night. Also note that on Friday, we'll be celebrating the 17th birthday of the Matroska format.</p>
+
+<h2>Where</h2>
+<p><a href="http://www.osaarchivum.org/">Vera and Donald Blinken Open Society Archives</a>, Arany János u. 32, 1051 Budapest, Hungary</p>
+
+<p>
+{% image '@NoTimeToWaitBundle/Resources/public/img/OSAatCEU_BudapestHungary_02.jpg' %}
+  <img src="{{ asset_url }}" alt="Open Society Archives at CEU in Budapest, Hungary" class="img-responsive center-xs" title="Open Society Archives at CEU in Budapest, Hungary">
+{% endimage %}
+<br />
+<em>Photo Credit: <a href="">Vera and Donald Blinken Open Society Archives</a></em></p>
```

Comment:
Bad copy/paste, I changed the credit name (please confirm it).
Do you have an URL for the credit?